lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -406,11 +409,6 @@ public static function getCorrectFileName($fileName)
         $fileInfo = pathinfo($fileName);
         $fileInfo['extension'] = $fileInfo['extension'] ?? '';
 
-        // account for excessively long filenames that cannot be stored completely in database
-        if (strlen($fileInfo['basename']) > 90) {
```

Comment:
Seems like we can change it to 255 instead of removing this change at all. What do you thing?

Reply:
If filename + extension has 255 characters and the directory is for example /a/b/c/ (7 characters) then in total we'll have 262 characters so it isn't going to be saved all 262 characters in varchar(255).

I think it's better we have in total 255 characters (directory + filename + extension).

Reply:
Ok, but I'm pretty sure some functionality might use this class, so it will allow saving files even with longer filenames.

I think we need to put here some limit quite close to 255, maybe 200 symbols? I think it should be enough for most cases. What do you thing?